cds/modules/maintenance/cli.py

```diff
@@ -38,13 +37,57 @@
     create_subformat,
 )
 from cds.modules.records.api import CDSVideosFilesIterator
+from cds.modules.deposit.api import Video
 from cds.modules.records.resolver import record_resolver
+from cds.modules.records.minters import doi_minter
+from cds.modules.deposit.tasks import datacite_register
+from invenio_db import db
+from cds.modules.records.permissions import is_public
+from invenio_indexer.api import RecordIndexer
+
 
 
 def abort_if_false(ctx, param, value):
     if not value:
         ctx.abort()
 
+@click.command()
+@click.option("--recid", "recid", help="ID of the video record", default=None)
+@with_appcontext
+def create_doi(recid):
+    if not recid:
```

Comment:
I think you can avoid that with marking the `@click.option` as required. See https://click.palletsprojects.com/en/stable/options/#basic-value-options